src/cards.json

```diff
@@ -0,0 +1,82 @@
+[
+{
+    "word": "Good-bye",
+    "translation": "Au revoir",
+    "isSelected": false
```

Comment:
more reading on MAP => https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map

I bring it up because I was thinking about how we were going to match objects. There's pros and cons, but with Map, we can just find the corresonding object by calling the word itself. Con's would be that every word needs to be unique. 

What do you think?